package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@jsmney/minitools",
+  "version": "2.3.0",
```

Comment:
FYI you wouldn't usually go all the way up to version 2 of the package in 3 hours - it indicates major, sometimes breaking (i.e. non-backwards compatible), changes to the package. Consider bumping only patch and/or minor versions during development.